backend/routes/products.js

```diff
@@ -2,43 +2,42 @@ const express = require("express");
 const router = express.Router();
 const Product = require("../models/Product");
 
-// return list of products
+// return list of products (get request)
```

Comment:
remove comment,
good code, which is short does not need useless comments.

you understand its a get request because of router.get, in addition you can understand you are returning the products as this is products/ route which does not need these comments